packages/app/src/cli/services/app/select-app.ts

```diff
@@ -47,7 +51,7 @@ export function remoteAppConfigurationExtensionContent(
   flags: Flag[],
 ) {
   let remoteAppConfig: {[key: string]: unknown} = {}
-  const configSpecifications = specifications.filter((spec) => spec.experience === 'configuration')
+  const configSpecifications = specifications.filter((spec) => spec.uidStrategy !== 'uuid')
```

Comment:
can't we use `spec.uidStrategy === 'single'` here? That will include the `webhooks` specs which already has the transform code for this and should be enough no?

Reply:
I tested by using `=== 'single'` here and the link command works fine, should we change it then?